apps/api/src/imbi/api/plugins/lifecycle_dispatch.py

```
@@ -157,9 +204,9 @@ async def dispatch_lifecycle(
     previous_project_slug: str | None = None,
     previous_project_type_slugs: list[str] | None = None,
     previous_team_slug: str | None = None,
-    project_name: str | None = None,
-    project_description: str | None = None,
-    project_ui_url: str | None = None,
+    project_name: str | None | _Unset = _UNSET,
```

Ruff (0.16.1)

[warning] `None` not at the end of the type union.

Move `None` to the end of the type union

(RUF036)